### DISCUSSION TITLE
New version based on my own EasyMDE

### DISCUSSION THREAD

**Discussion by erossini** [erossini/BlazorMarkdownEditor#35] [Announcements]
With the new version of the [Markdown Editor for Blazor](https://puresourcecode.com/javascript/new-markdowneditor-components-for-javascript-and-blazor/), I started to use my own implementation of the [EasyMDE](https://github.com/erossini/EasyMarkdownEditor) because this JavaScript library is not maintained anymore.

**Comment by webluke** [comment]
Have you looked into using the most up to date version from Lonaru . They are on v2.18.0 https://github.com/Ionaru/easy-markdown-editor
